src/CandidateCard.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+
+function CandidateCard({candidate, index}) {
+    const handleVoteCount= async (candidate)=> {
+        await axios.get(
+             `http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/vote?id=${candidate.id}` 
+         )
+         .then ((response)=>{
+             alert(candidate.name + '님에게 투표 완료!');
+         })
+         .catch ((error)=> {
+             alert('투표 실패');
+         });
+    }
```

Comment:
1. 현재 GET통신이후 비동기처리방식이 Promise 방식으로 이루어지고 있어서 async await 는 필요없을 것 같아요!!
2. response와 error는 안 쓰이고 있어서 지워주셔도 좋을 것 같아요~
```suggestion
    const handleVoteCount= (candidate)=> {
        axios.get(
             `http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/vote?id=${candidate.id}` 
         )
         .then (()=>{
             alert(candidate.name + '님에게 투표 완료!');
         })
         .catch (()=> {
             alert('투표 실패');
         });
    }
```